src/Umbraco.Cms.Api.Management/Controllers/Security/BackOfficeController.cs

```diff
@@ -192,16 +200,35 @@ public async Task<IActionResult> Signout(CancellationToken cancellationToken)
     /// </summary>
     /// <param name="provider"></param>
     /// <returns></returns>
-    [HttpPost("link-login")]
+    [HttpGet("link-login")]
+    [AllowAnonymous]
     [MapToApiVersion("1.0")]
-    public IActionResult LinkLogin(string provider)
+    public async Task<IActionResult> LinkLogin(string provider)
     {
+        var cookieAuthenticatedUserAttempt =
+            await HttpContext.AuthenticateAsync(Constants.Security.BackOfficeAuthenticationType);
```

Comment:
We need to circle back to this once we've figure out a way to perform linking that does not depend on cookies.